src/actions/sortListActions.js

```diff
@@ -0,0 +1,33 @@
+import { SET_SORT } from './actionTypes';
+import { storage } from '../components/Dashboard/SortList/sortUtils';
+
+const setSort = (key, field, direction) => dispatch => {
+  storage.set(key, {
+    field,
+    direction,
+  });
+
+  dispatch({
+    type: SET_SORT,
+    key,
+    field,
+    direction,
+  });
+};
+
+const loadSort = key => dispatch => {
+  const data = storage.get(key) || {};
+  const { field, direction } = data;
+
+  dispatch({
+    type: SET_SORT,
```

Comment:
I can see only one type in two functions, which makes one of those redundant. Use dynamic dispatching techniques and refactor duplicate code